pkgs/development/compilers/terra/default.nix

```diff
@@ -1,68 +1,87 @@
-{ lib, stdenv, fetchurl, fetchFromGitHub
-, llvmPackages, ncurses, lua
-}:
+{ lib, stdenv, fetchurl, fetchFromGitHub, llvmPackages, ncurses, cmake, libxml2
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, llvmPackages, ncurses, cmake, libxml2
```

Reply:
Not needed anymore.

Reply:
Got this fixed in [`c49d9ad` (#125896)](https://github.com/NixOS/nixpkgs/pull/125896/commits/c49d9adc385ccff57ef020456f3700fe6878619d).